core/blockchain_test.go

```diff
@@ -1891,9 +1891,13 @@ func TestLowDiffLongChain(t *testing.T) {
 	db := rawdb.NewMemoryDatabase()
 	genesis := (&Genesis{BaseFee: big.NewInt(params.InitialBaseFee)}).MustCommit(db)
 
+	//Using TempTriesInMemory variable instead of DefaultTempInTries because changing the
+	//value of DefaultTempInTries to 1024 is failing the test.
+	TempTriesInMemory := 128
```

Comment:
Not a temporary fix, We need to change the value of TriesInMemory to 1024, But these test fail with 1024 value because in these tests they are trying  to import the DefaultTriesInMemory(value) chain.
But please take one look into it.

Reply:
Okay